index.bs

```diff
@@ -1677,9 +1677,14 @@ Origin of Blob URLs</h4>
 <a>Blob URLs</a> are created using <code>URL.{{URL/createObjectURL()}}</code>,
 and are revoked using <code>URL.{{URL/revokeObjectURL()}}</code>.
 The <dfn lt="origin of a Blob URL|Blob URL's origin">origin of a Blob URL</dfn> must be the same as the [=environment settings object/origin=]
-specified by the <a>incumbent settings object</a>
+specified by the <a>current settings object</a>
 at the time the method that created it was called.
 
+WARNING: there is currently some confusion between the generic definition of the
```

Comment:
I don't think "WARNING" is a thing with special meaning in bikeshed. Probably `Advisement:` or `Issue:` makes the most sense?